autometa/common/external/bowtie.py

```diff
@@ -128,7 +128,7 @@ def align(db, sam, fwd_reads=None, rev_reads=None, se_reads=None, cpus=0, **kwar
     for flag, reads in zip(["-1", "-2", "-U"], [fwd_reads, rev_reads, se_reads]):
         if reads:
             reads_provided = True
-            params.append(f'{flag} {",".join(reads)}')
+            params.append(f"{flag} {reads}")
```

Comment:
This is just a formatting nitpick.

```suggestion
            listed_reads = ", ".join(reads)
            params.append(f"{flag} {listed_reads}")
```